script/main.py

```diff
@@ -4,15 +4,25 @@
 
 
 def fetch_argument():
-  parser = argparse.ArgumentParser()
+  parser = argparse.ArgumentParser()   
+  parser.add_argument('--driver', help='entry driver')
   parser.add_argument("url", help="entry page url")
-  args = parser.parse_args()
-  return(args.url)
+  args = parser.parse_args()     
+  return(args.driver, args.url)
 
 
-def main():
-  driver = webdriver.Chrome(executable_path="bin/chromedriver")
-  driver.get(fetch_argument())
+def create_driver(name):
+  if name  == 'chrome':
+    driver = webdriver.Chrome(executable_path="bin/chromedriver")
+  else:
+    driver = webdriver.Firefox(executable_path="bin/geckodriver.exe")
+  return(driver)
 
 
+def main():  
+  (driver, url) = fetch_argument()
```

Comment:
BTW you don't need parentheses here. Technically it is correct but most of the time people don't do it in python.